src/backend/app/modules/Device/DeviceService.php

```diff
@@ -18,7 +18,9 @@ public function __construct(
     ) {}
 
     public function make($deviceData): Device {
-        $result = $this->device->newQuery()->create([
+        // This method is used to create a new instance of the model
+        // without saving it to the database immediately.
```

Comment:
I don't think you need that comment 😉 I think it can be deducted by convention that the `make` function doesn't actually resist a record to the database (this is what the `save` function is for). And `create` being an alias for `make` + `save`.

I'd say it's actually a massive bug that the `make` function of `DeviceService` calls `create`. Great find btw 😉